New endstop holder for PhotoelectricBarriers.

WartyTowel opened the pull request:
Here are the endstop holder for photoelectric barriers.

WartyTowel committed:
New endstop holder for PhotoelectricBarriers.

pkucmus commented:
Thank you! I am still trying to figure out how to take contributions but I think you could add a step directory and you step files there, I can add those to the documentation. Also please let me know what to put under credits (link to you page of something like that).

WartyTowel committed:
Added the step files.

WartyTowel commented:
Hi, you are welcome! I added the step files and I removed the blank (" ") in the filenames of the stl files. I hope this pull request has the new files now too. Please accept it if not, I will create a new one than. 
I do not have a page ore something like that to revere to. You can simply mention "WartyTowel", I would appreciate that. :)
